src/codegen/properties/functions.py

```diff
@@ -0,0 +1,75 @@
+"""Structure for storing function metadata from scrapigen."""
+from codegen.properties.adaptor_parameter import AdaptorParameter
+from codegen.properties.function_parameters import FunctionsParameter
+from codegen.properties.parameter import Parameter
+
+
+class Functions:
+    """Structure for storing function metadata from scrapigen."""
+
+    def __init__(self, function_name, function_metadata):
+        """Structure for storing function metadata from scrapigen."""
+        self._function_name = function_name
+        self._c_function_name = function_metadata["c_function_name"]
```

Comment:
`c_function_name` no longer exists.

```suggestion
        if "cname" in function_metadata:
            assert function_metadata["cname"].startswith("DAQmx")
            self._c_function_name = function_metadata["cname"][5:]
        else:
            self._c_function_name = function_name
```